docs/cli/ctl.md

```diff
@@ -0,0 +1,369 @@
+---
+title: "wash ctl"
+draft: false
+sidebar_position: 6
+description: "wash ctl command reference"
+--- 
+
+This subcommand will help you to interact with the wasmCloud control interface. You may perform operations such as retrieving information about the lattice, enable provisioning or deprovisioning of entities in the lattice, creating links or updating items in the lattice. Following are the subcommands available under `wash ctl`
+
+- `get`
+- `link`
+- `start`
+- `stop`
+- `update`
+- `scale`
+
+## `get`
+wash get retrives information about the lattice. The following subcommands will help you retrieve different information available to query:
+
+:::caution
+This subcommand will be deprecated in furture versions. Please use `wash get` instead.
+:::
+
+### `claims`
+This subcommand fetches all the known claims inside the lattice.
+
+#### Usage
+```
+wash ctl get claims
+```
+
+### `hosts`
+This subcommand fetches all the responsive hosts in the lattice. You may also view the friendly name for all the hosts here.
+
+#### Usage
+```
+wash ctl get hosts
+```
+
+### `inventory`
+This subcommand retrieves the inventories of all the hosts in the lattice. To retrieve the inventory of a single host, you may pass the host-id of the desired host.
+
+#### Usage
+```
+wash ctl get inventory
+wash ctl get inventory <your-host-id>
+```
+
+#### Options
+These options can be used for all of the `wash ctl get` subcommands. 
+
+`--output` (Alias `-o`) Specify output format (text or json) [default: text]
```

Comment:
Same thing with global options here

Reply:
Yea. Since these options were repeated for all the subcommands of `wash ctl get`, I put them together